internal/provider/resources/service_account.go

```diff
@@ -93,18 +104,13 @@ func (r *ServiceAccountResource) Schema(_ context.Context, _ resource.SchemaRequ
 				Description: "Date and time that the service account was last updated in RFC 3339 format",
 			},
 			"account_id": schema.StringAttribute{
+				Optional:    true,
 				Computed:    true,
 				CustomType:  customtypes.UUIDType{},
 				Description: "Account UUID, defaults to the account set in the provider",
 			},
-			// @TODO: Do we actually need 'account_role_id' defined as an attribute here?
-			// or just only one of account_role_id or account_role_name?
-			"account_role_id": schema.StringAttribute{
```

Comment:
account_role_id will be a request parameter upon Create, but since it doesn't get returned from the API (nor does the id have have meaning for the practitioner here), we'll omit it from the resource schema

in a followup, we'll use an optional `account_role_name` attribute to do an `account_role` lookup to find the id + pass it into the Create call, which will remain under the hood / out of view from the practitioner